contracts/ronin/RoninGovernanceAdmin.sol

```diff
@@ -3,18 +3,20 @@ pragma solidity ^0.8.0;
 
 import "../extensions/bridge-operator-governance/BOsGovernanceProposal.sol";
 import "../extensions/sequential-governance/GovernanceProposal.sol";
-import "../extensions/collections/HasValidatorContract.sol";
+import "../extensions/collections/HasContract.sol";
 import "../extensions/GovernanceAdmin.sol";
 import "../libraries/EmergencyExitBallot.sol";
 import "../libraries/ErrorHandler.sol";
+import "../interfaces/IRoninTrustedOrganization.sol";
+import "../interfaces/validator/IRoninValidatorSet.sol";
 import "../interfaces/IRoninGovernanceAdmin.sol";
 
 contract RoninGovernanceAdmin is
```

Comment:
Note for deployer: need to redeploy Ronin GA

Reply:
noted.